### ISSUE TITLE
Inputs failed to validate in the `Task` resource

### ISSUE THREAD

**Issue by LouisAmon** [pulumi/pulumi-snowflake#126]
### What happened?

Importing an existing `Task` gives the following error:
```log
warning: One or more imported inputs failed to validate. This is almost certainly a bug in the `snowflake` provider. The import will still proceed, but you will need to edit the generated code after copying it into your program.
warning: snowflake:index/task:Task resource 'usages' has a problem: Conflicting configuration arguments: "schedule": conflicts with after. Examine values at 'Task.Schedule'.
warning: snowflake:index/task:Task resource 'usages' has a problem: Conflicting configuration arguments: "after": conflicts with schedule. Examine values at 'Task.After'.
```

### Steps to reproduce

```
pulumi import snowflake:index/task:Task usages 'DATABASE|SCHEMA|TASK'
```

Then `pulumi up`

### Expected Behavior

We should be able to run `pulumi up` straight after import with no error

### Actual Behavior

Running pulumi up after import gives the following error:
```
    error: 1 error occurred:
        * updating urn:pulumi:prod::snowflake::snowflake:index/task:Task::usages: 1 error occurred:
        * error retrieving root task MY_TASK: failed to locate the root node of: []: sql: no rows in result set
```

It seems like it's trying to fetch a task named "[]" (which doesn't exist), whereas my task is actually a root task being run on a schedule and not after a given task.

Maybe `after` has a default value of [] and is messing around ?

### Versions used

Dependencies:
NAME              VERSION
black             22.6.0
flake8            4.0.1
pip               22.0.4
pulumi-snowflake  0.10.0
pylint            2.14.4
pyright           1.1.256

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by jkodroff** [comment]
@LouisAmon Thanks for submitting this issue. Could you provide the task definition, replacing the actual SQL query with `SELECT 0` so as to provide a minimal and portable example? I'm not seeing this behavior with the following task, and I'm not especially familiar with Snowflake:

```sql
create or replace task jkodroff_test warehouse=compute_wh schedule = '60 minute' as select 1
```

**Comment by samanthawalter** [comment]
I ran into this too. It seems to be an [issue with terraform-provider-snowflake](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1071) used by pulumi-snowflake. It was resolved in v0.37.0 of the terraform-provider-snowflake. Upgrading to at least [v0.11.0 of pulumi-snowflake](https://github.com/pulumi/pulumi-snowflake/blob/v0.11.0/provider/go.mod#L12) should solve the issue.
